packages/react-app/src/components/ConfirmTransferModal.jsx

```diff
@@ -34,6 +36,7 @@ export const ConfirmTransferModal = ({ isOpen, onClose }) => {
   const fromUnit = fromToken.symbol + (isxDai ? ' on xDai' : '');
   const toAmt = formatValue(toAmount, toToken.decimals);
   const toUnit = toToken.symbol + (isxDai ? '' : ' on xDai');
+  const isERC20Dai = toToken.symbol === 'DAI';
```

Comment:
In either cases, should we do a hard-coded address check instead of symbol check?